Analysis/include/SampleDescriptor.h

```diff
@@ -20,11 +20,14 @@ struct AnalyzerSetup {
     Period period;
     SignalMode mode;
     QCDmethod qcd_method;
+    EventRegion qcd_shape;
     DiscriminatorWP tauID_wp;
     std::vector<std::string> final_variables;
     std::vector<double> pt_sel_bins;
-    bool apply_mass_cut{false}, apply_os_cut{true}, apply_iso_cut{true};
+    bool apply_mass_cut{false}, apply_os_cut{true};
     bool use_kinFit{false}, use_svFit{false}, applyTauID{true};
+    DiscriminatorWP tau_iso_wp;
+    TauIdDiscriminator tau_iso_disc;
```

Comment:
`tau_iso_wp` and `tau_iso_disc` are declared but never used. Or remove them or use them somewhere in the analyzer.